projects/luxon-angular/src/lib/formatting/date-time-to-locale-string.pipe.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { DateTime, LocaleOptions } from 'luxon';
+import { DateTimeToLocaleStringPipe } from './date-time-to-locale-string.pipe';
+
+describe('dateTimeToLocaleString', () => {
+  const format = DateTime.DATETIME_FULL;
+  const localeOptions: LocaleOptions = {locale: 'en-US'}
+  const pipe = new DateTimeToLocaleStringPipe();
+
+  describe('#transform', () => {
+    it('transforms an empty input into null', () => {
```

Comment:
Add another test for null format

Reply:
Thanks for your feedback. The thing is I don't know how to porperly assert the output, if I cannot specify the locale. Because depending on the system the output will be different.

e.g. if format is null, the output will either be MM/DD/YYYY or DD.MM.YYYY depending on the locale

Do you have an idea how to write this test?

Reply:
In [other tests](https://github.com/dstelljes/luxon-angular/blob/main/projects/luxon-angular/src/lib/formatting/date-time-to-format.pipe.spec.ts#L16) we’ve used regular expressions to account for possible timezone differences, but yeah, that may not be possible here. Would using [`Date.toLocaleString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString) to get the expected output work?